llvm/test/CodeGen/AArch64/sve-streaming-mode-cvt-fp-int-fp.ll

```diff
@@ -1,22 +1,32 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -force-streaming-compatible  < %s | FileCheck %s
-; RUN: llc -force-streaming-compatible -mattr=+sme2p2  < %s | FileCheck %s --check-prefix=USE-NEON-NO-GPRS
-; RUN: llc < %s | FileCheck %s --check-prefix=USE-NEON-NO-GPRS
+; RUN: llc -mattr=+sve -force-streaming-compatible < %s | FileCheck %s
+; RUN: llc -mattr=+sme -force-streaming < %s | FileCheck %s
```

Comment:
Based on my comment above and because the new RUN lines are code quality tests rather than functionality tests I think you should test `-mattr=+sve2` and `-mattr=+sme,+fullfp16` which I believe will match the current output without needing the extra extend/round handling within the combine.

Reply:
This was fixed with 6f013dbced67948119fe9ca71336f0284975ba4f, so just `+sme` works (no need to add `+fullfp16 ` anymore)